icons/figma.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you run the SVG through an optimizer (e.g. [SVGOMG](https://jakearchibald.github.io/svgomg/)) and update it, e.g.:

```suggestion
<svg aria-labelledby="simpleicons-figma-icon" role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title id="simpleicons-figma-icon">Figma icon</title><path d="M12 12c0-1.0609.4214-2.07828 1.1716-2.82843C13.9217 8.42143 14.9391 8 16 8c1.0609 0 2.0783.42143 2.8284 1.17157C19.5786 9.92172 20 10.9391 20 12s-.4214 2.0783-1.1716 2.8284C18.0783 15.5786 17.0609 16 16 16c-1.0609 0-2.0783-.4214-2.8284-1.1716C12.4214 14.0783 12 13.0609 12 12zM4 20c0-1.0609.42143-2.0783 1.17157-2.8284C5.92172 16.4214 6.93913 16 8 16h4v4c0 1.0609-.4214 2.0783-1.1716 2.8284C10.0783 23.5786 9.06087 24 8 24c-1.06087 0-2.07828-.4214-2.82843-1.1716C4.42143 22.0783 4 21.0609 4 20zM12 0v8h4c1.0609 0 2.0783-.42143 2.8284-1.17157C19.5786 6.07828 20 5.06087 20 4s-.4214-2.07828-1.1716-2.82843C18.0783.421427 17.0609 0 16 0h-4zM4 4c0 1.06087.42143 2.07828 1.17157 2.82843C5.92172 7.57857 6.93913 8 8 8h4V0H8C6.93913 0 5.92172.421427 5.17157 1.17157 4.42143 1.92172 4 2.93913 4 4zM4 12c0 1.0609.42143 2.0783 1.17157 2.8284C5.92172 15.5786 6.93913 16 8 16h4V8H8c-1.06087 0-2.07828.42143-2.82843 1.17157C4.42143 9.92172 4 10.9391 4 12z"/></svg>
```